tasks/copy_files_to_archive/copy_files_to_archive.py

```diff
@@ -4,29 +4,46 @@
 Description:  Lambda function that copies files from one s3 bucket
 to another s3 bucket.
 """
-
+import json
+import logging
 import os
 import time
-import logging
-from typing import Any, List, Dict, Optional
+from enum import Enum
+from typing import Any, List, Dict, Optional, Union
 
 import boto3
+# noinspection PyPackageRequirements
+import database
+# noinspection PyPackageRequirements
 from botocore.client import BaseClient
+# noinspection PyPackageRequirements
 from botocore.exceptions import ClientError
-import requests_db
+
+
+class RequestMethod(Enum):
+    POST = 'post'
+    PUT = 'put'
+
+
+OS_ENVIRON_DB_QUEUE_URL_KEY = 'DB_QUEUE_URL'
 
 # These will determine what the output looks like.
 FILE_SUCCESS_KEY = 'success'
-FILE_SOURCE_KEY_KEY = 'source_key'
 FILE_ERROR_MESSAGE_KEY = 'err_msg'
-FILE_REQUEST_ID_KEY = 'request_id'
-FILE_SOURCE_BUCKET_KEY = 'source_bucket'
-FILE_TARGET_BUCKET_KEY = 'target_bucket'
 
-# These are tied to the database schema.
-JOB_REQUEST_ID_KEY = 'request_id'
-JOB_ARCHIVE_BUCKET_DEST_KEY = 'archive_bucket_dest'
-JOB_JOB_STATUS_KEY = 'job_status'
+# These are tied to the input schema.
```

Comment:
These are certainly not meant as a replacement. Once things are actually tested and finalized I'll add something, but we're seeing a lot of churn.

Reply:
I'm loathe to remove these consts, since even our most recent conversation discussed changing interaction format with the paging and such, and I don't want to rely on whether I type a plain-text-string correctly.